website/pages/docs/post-processors/manifest.mdx

```diff
@@ -38,17 +36,37 @@ post-processors such as Docker and Artifice.
 
 @include 'post-processor/manifest/Config-not-required.mdx'
 
-- `keep_input_artifact` (boolean) - Unlike most other post-processors, the
```

Comment:
Can we do a follow-up PR where we re-add this as a note rather than a configuration option? I think it's important to have documented.

Reply:
Yes! I'll do it